agent/action/mount_disk.go

```diff
@@ -81,12 +79,3 @@ func (a MountDiskAction) Resume() (interface{}, error) {
 func (a MountDiskAction) Cancel() error {
 	return errors.New("not supported")
 }
-
-func (a MountDiskAction) pruneNil(hints []interface{}) []interface{} {
-	for i := len(hints) - 1; i >= 0; i-- {
-		if hints[i] == nil {
-			hints = append(hints[:i], hints[i+1:]...)
-		}
-	}
-	return hints
-}
```

Comment:
I assume `pruneNil` is not called anywhere. Nice job cleaning up!

Reply:
Yup, quite a bit of dead code. Mostly stuff that was replaced by `sigar` but never removed.